tests/br_full/run.sh

```diff
@@ -53,6 +53,17 @@ run_br --pd $PD_ADDR backup full -s "local://$TEST_DIR/$DB-lz4" --concurrency 4
 export GO_FAILPOINTS=""
 size_lz4=$(du -d 0 $TEST_DIR/$DB-lz4 | awk '{print $1}')
 
+echo "backup with tikv error start..." 
+export GO_FAILPOINTS="github.com/pingcap/br/pkg/backup/tikv-rw-error=return(\"tikv read or write error\")"
+run_br --pd $PD_ADDR backup full -s "local://$TEST_DIR/$DB-tikverr" --concurrency 4 &
+pid=$!
+export GO_FAILPOINTS=""
+sleep 15
```

Comment:
Consider add a check for whether the new error message was printed here?

Reply:
ok